cmd/agent/main.go

```diff
@@ -12,20 +12,29 @@ import (
 	"github.com/andrei-cloud/go-devops/internal/collector"
 )
 
+var (
+	buildVersion string
```

Comment:
```suggestion
	buildVersion = "N/A"
```
может так? и не надо проверку далее